script.sh

```diff
@@ -18,9 +18,6 @@ then
 fi
 echo "Script is being run as root."
 
-echo "Running apt-get update"
-apt-get update
```

Comment:
Why would we remove this? We need to update repositories on first-time use, otherwise installing packages may not work due to outdated metadata and information.